lua/utils.lua

```diff
@@ -1,29 +1,91 @@
 local M = {}
-local loop = vim.loop
+local uv = vim.loop
+local message = {}
```

Comment:
We should try and move the scope of this down into the `spawn` function. That way, we don't get old messages displayed in the `showError` function.